src/depfile_parser.cc

```diff
@@ -1,4 +1,4 @@
-/* Generated by re2c 0.13.5 */
+/* Generated by re2c 0.16 */
```

Comment:
Can you either generate the file with re2c 0.13.5 or add a separate commit which updates the file in Git from re2c 0.13.5 to 0.16?